contracts/integrations/test/utils/deployment_mananger.ts

```diff
@@ -441,7 +479,7 @@ export class DeploymentManager {
         };
     }
 
-    private constructor(
+    constructor(
```

Comment:
When would a user want to call this? The reason I originally made it private was that it would encourage users to just use `deployAsync` instead of rolling their own deployment.

Reply:
oh yeah this should be protected –– `DydxDeploymentManager` calls `super(...)` in its constructor

Reply:
Ok, I see why you changed it. Yeah, making it `protected` makes sense.